firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -336,10 +332,6 @@ class Meta:
                 "translations",
                 "part_of_speech",
                 "pronunciations",
-                "split_chars",
```

Comment:
Removing these fields instead of commenting due to sonar. But leaving the methods to compute these fields. Should be added back with a parameter as a part of [FW-4931](https://firstvoices.atlassian.net/browse/FW-4931)